src/transformers/integrations/ggml.py

```diff
@@ -181,6 +181,7 @@
         "ggml.unknown_token_id": "unk_token_id",
         "ggml.padding_token_id": "pad_token_id",
         "ggml.add_space_prefix": "add_prefix_space",
+        "chat_template": "chat_template",
```

Comment:
It looks like `tokenizer_config` is not used at all. Is this something we can remove ? This is confusing to have `tokenizer` and `tokenizer_config` with practically the same mapping. For example, I expected that `chat_template` would work since it's in `tokenizer_config`

Reply:
I think `tokenizer_config` is designed to match the content of general tokenizer_config_file. I have updated the code to add template through `tokenizer_config`.